Sources/sentry/include/SentryApp.h

```diff
@@ -5,4 +5,28 @@
 
 #include <sentry.h>
 
-#endif
\ No newline at end of file
+#ifdef _WIN32
+#include <atomic>
+
+// The type of swift::swift_getFatalErrorMessageBuffer
+using swift_getFatalErrorMessageBuffer_t = std::atomic<const char *> *(*)();
+
+inline void *getFatalErrorMessageHandle() {
+  HMODULE hSwiftCore = GetModuleHandleA("swiftCore.dll");
+  if (!hSwiftCore) { return nullptr; }
+  swift_getFatalErrorMessageBuffer_t pGetBuf =
```

Comment:
We should document the expectation that the export might not exist because it was introduced in a more recent version of the toolchain.

Reply:
Done